data/json/recipes/recipe_food.json

```diff
@@ -4142,6 +4142,46 @@
       [ [ "water", 1 ], [ "water_clean", 1 ] ]
     ]
   },
+  {
+    "type": "recipe",
+    "result": "pizza_supreme",
+    "category": "CC_FOOD",
+    "subcategory": "CSC_FOOD_MEAT",
+    "skill_used": "cooking",
+    "difficulty": 3,
+    "time": "30 m",
+    "book_learn": [ [ "cookbook_italian", 3 ] ],
+    "qualities": [ { "id": "COOK", "level": 2 } ],
+    "tools": [ [ [ "surface_heat", 8, "LIST" ] ] ],
+    "components": [
+      [ [ "flour", 30 ] ],
+      [
+        [ "meat_red", 2, "LIST" ],
+        [ "can_chicken", 2 ],
+        [ "meat_salted", 2 ],
+        [ "dry_meat", 2 ],
+        [ "fish", 2 ],
+        [ "fish_canned", 2 ],
+        [ "can_tuna", 2 ],
+        [ "salted_fish", 2 ],
+        [ "rehydrated_fish", 2 ],
+        [ "dry_fish", 2 ]
+      ],
+      [
+        [ "veggy_any", 2, "LIST" ],
+        [ "mushroom", 2 ],
+        [ "dry_mushroom", 2 ],
+        [ "morel_cooked", 2 ],
+        [ "tofu", 2 ],
+        [ "dry_tofu", 2 ],
```

Comment:
Perhaps the tofus would be better put on the meat list? In terms of recipe, they're more appropriate there.